assets/js/blocks/collection-filters/inner-blocks/attribute-filter/style.scss

```diff
@@ -1,3 +1,7 @@
+//  Import styles we need to render the checkbox list and checkbox control.
+@import "../../../../../../packages/components/checkbox-list/style";
```

Comment:
As mentioned in the description we need to explicitly load these styles in case they haven't been loaded by another block. We should move checkbox list into a reusable component like we did with dropdown to improve this situation. This is the same approach I took in the stock filter.